build/gen_stub.php

```diff
@@ -1810,24 +1822,29 @@ private function getConstDeclaration(string $code, EvaluatedValue $value): strin
         $constName = str_replace('\\', '\\\\', $this->name->__toString());
         $constValue = $value->value;
 
+        $flags = "CONST_CS | CONST_PERSISTENT";
```

Comment:
Off: would it make sense to get rid of `CONST_CS` in PHP 8.2? As it's not used anymore...